easybuild/easyconfigs/g/GPAW/GPAW-21.6.0-intel-2020b-ASE-3.22.0.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '21.6.0'
+_aseversion = '3.22.0'
+versionsuffix = '-ASE-' + _aseversion
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+toolchainopts = {'usempi': True, 'openmp': False}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('GPAW-20.1.0-Add-Easybuild-configuration-files.patch', 1),
+]
+checksums = [
+    '5bb805bf514a7b04e3fdfce6f7864d150032badc5cd2805c57513af982d7a9cc',  # gpaw-21.6.0.tar.gz
+    # GPAW-20.1.0-Add-Easybuild-configuration-files.patch
+    'a12440bf63af70b891a63989b0f048bb8ebf4f60499020ea09259937f04cd042',
+]
+
+# The intel version is not built with libvdwxc support, as that library is incompatible with MKL.
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('ASE', _aseversion),
+    ('libxc', '4.3.4'),
+    ('GPAW-setups', '0.9.20000', '', True),
+]
+
+prebuildopts = 'GPAW_CONFIG=doc/platforms/Linux/EasyBuild/config_intel.py'
+preinstallopts = prebuildopts
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+# required because we're building a Python package using Intel compilers on top of Python built with GCC.
+check_ldshared = True
```

Comment:
Same here, please remove

Reply:
@boegel 
Should I just remove ``check_ldshared`` or also ``use_pip`` and ``sanity_pip_check`` ?

Reply:
just check_ldshared

Reply:
Indeed, `use_pip` and `sanity_pip_check` are not enabled by default yet, we plan to do that with EasyBuild v5.0.

Reply:
Thanks @boegel and @Micket, I have removed check_ldshared now.